specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2021-06-01-preview/examples/ServerCreate.json

```diff
@@ -0,0 +1,136 @@
+{
+  "parameters": {
+    "serverName": "pgtestsvc4",
+    "resourceGroupName": "testrg",
+    "api-version": "2021-06-01-preview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "parameters": {
+      "location": "westus",
+      "sku": {
+        "tier": "GeneralPurpose",
+        "name": "Standard_D4s_v3"
+      },
+      "properties": {
+        "administratorLogin": "cloudsa",
+        "administratorLoginPassword": "password",
+        "version": "12",
+        "availabilityZone": "1",
+        "createMode" : "Create",
+        "storage": {
+          "storageSizeGB": 512
+        },
+        "backup": {
+          "backupRetentionDays": 7,
+          "geoRedundantBackup": "Disabled"
+        },
+        "network": {
+          "delegatedSubnetResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testrg/providers/Microsoft.Network/virtualNetworks/test-vnet/subnets/test-vnet-subnet",
+          "privateDnsZoneArmResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourcegroups/testrg/providers/Microsoft.Network/privateDnsZones/test-private-dns-zone.postgres.database.azure.com"
+        },
+        "highAvailability": {
+          "mode": "ZoneRedundant"
+        }
+      },
+      "tags": {
+        "ElasticServer": "1"
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "sku": {
+            "name": "Standard_D4s_v3",
+            "tier": "GeneralPurpose"
+        },
+        "properties": {
+            "fullyQualifiedDomainName": "c7d7483a8ceb.test-private-dns-zone.postgres.database.azure.com",
```

Comment:
shouldn't it be `pgtestsvc4.postgres.database.azure.com` ??

Reply:
Thanks. This is the scenario of VNet injection. So there won't be a public IP for server. Instead, the private DNS zone will be used to access server. Tao should know this better. The differences between public IP server and VNet injection server are shown in ServerGet.json and ServerGetWithVnet.json

Reply:
Thanks a lot Tiger for checking this!